rules/PSR4/NodeManipulator/FullyQualifyStmtsAnalyzer.php

```diff
@@ -65,7 +65,10 @@ public function process(array $stmts): void
                 return null;
             }
 
-            return new FullyQualified($name);
+            $fullyQualified = new FullyQualified($name);
+            $fullyQualified->setAttribute(Attributekey::PARENT_NODE, $parentNode);
+
+            return $fullyQualified;
```

Comment:
this parent attribute mirror is no longer needed, covered at https://github.com/rectorphp/rector-src/pull/2962/files#diff-66beba605a2f40ccb06a352cb8d23db9246969feb7bc25d624cb200f8ba6af12

Reply:
Dropped the commit.